src/main/resources/db/migration/nomis/data/R__9_0__SMOKE_TEST_USER.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO STAFF_MEMBERS
+  (STAFF_ID, ASSIGNED_CASELOAD_ID, WORKING_CASELOAD_ID, LAST_NAME, FIRST_NAME, MIDDLE_NAME, STATUS, SEX_CODE, BIRTHDATE) VALUES
+  (-30, 'LEI', 'LEI', 'SMOKE', 'TEST', null, 'ACTIVE', 'F', '1971-05-01');
+
+INSERT INTO STAFF_USER_ACCOUNTS
+  (USERNAME, STAFF_ID, STAFF_USER_TYPE, ID_SOURCE, WORKING_CASELOAD_ID) VALUES
+  ('SMOKE_TEST_USER', -30, 'GENERAL', 'USER', 'MUL');
```

Comment:
I needed a staff user for smoke_test_user to enable the release to be recorded against a staff user

Reply:
i wonder why we didn't need to do this for imprisonment status? I guess since that just updated the database rather than calling the real service ???

Reply:
re: i wonder why we didn't need to do this for imprisonment status? I guess since that just updated the database rather than calling the real service ???